### ISSUE TITLE
Load Only Fragment or Check if pjax is Already Instantiated

### ISSUE THREAD

**Issue by jackrugile** [defunkt/jquery-pjax#116]
Example of issue here: http://jackrugile.com/lab/pjax-v1/

Everything seems to be working fine at first (persistent animation on bottom just to make it clear whether the page is reloading or not), but pjax is getting bound multiple times. If you view the console, you can see the logs I have attached to start and end continue to add up as you navigate.

I specified a container and fragment, so I assumed that is all that would be pulled and run from my request, but is my JavaScript in the included footer supposed to be run as well? If that is the intended functionality, how can I check if pjax is already instantiated in my script?

**In short: I need to prevent pjax from being instantiated multiple times.**

I may be misunderstanding how pjax is fundamentally supposed to work. Let me know if you have a solution, if I am doing this all wrong, or if you need more information. Thanks!
